app/views/people/index.erb

```diff
@@ -0,0 +1,20 @@
+<p class='link'><%= link_to "Go back to task list manager", "/" %></p>
+<h1>We are TASK TEAM</h1>
+
+<%= image_tag("people.png", :alt => "team", :class => "face") %>
+
+<table>
+  <tr>
+    <th>Name</th>
+    <th>Incomplete tasks</th>
+    <th></th>
+  </tr>
+
+  <tr>
+    <% @people.each do |person| %>
+    <td><%= person[:name] %></td>
```

Comment:
I would recommend referring to the model fields with the `model.field` syntax rather than `model[:field]` syntax that is used here because it is easier to read.